bin/build-release.sh

```diff
@@ -31,42 +30,24 @@ fi
 
 umask 0022
 
-git clone $REPOPATH $TARGET
+git clone --depth 1 --branch $TAG $REPOPATH $TARGET
 cd $TARGET
-git checkout $TAG
-cd ..
 
-if [ ! -x "$TARGET/composer.phar" ]; then
-    curl -sS https://getcomposer.org/installer | php -- --install-dir=$TARGET
+if [ ! -x "$COMPOSER" ]; then
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/tmp
```

Comment:
/tmp might not be executable, but since this is only of concern to those building releases it's probably not an issue

Reply:
To my understanding, this doesn't count for script-files that start with a she-bang, because the actual interpreter (the php binary) is not on the noexec volume.

My next step, now that this script is cleaned up, is to migrate it to a Github Action, where composer will be available out-of-the-box